### ISSUE TITLE
+Weapon damage enchants increase druid damage in forms.

### ISSUE THREAD

**Issue by Fiskehatt** [LightsHope/server#264]
Apparently weapon damage enchants like http://db.vanillagaming.org/?spell=13937 still boost druids' melee  damage in both Cat and Bear form. I have met many people with the enchant, who claim it still works. This is an old bug that was already identified on Nostalrius:

https://forum.nostalrius.org/viewtopic.php?f=41&t=36377&start=30#p269579:
https://forum.nostalrius.org/viewtopic.php?f=41&t=36377&start=100#p287384

The enchants should not give any damage boost, similarly to +weapon damage sharpening stones:

http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-guides/4391-speed-up-molten-core.html:

> Casters can use Wizard Oil to increase their damage; melee DPS can use Sharpening Stones or Weightstones or Posions. Potions and Elixirs can also be used, but are a bit more expensive and do not last through death. Help provide them for your most effective players, but don’t insist on them unless you’re pushing hard to set a guild record… save them for content you really need the extra help on. One note for the druids out there — **only Elemental Sharpening Stones work in cat form, and they tend to be expensive to make.**

The enchants should probably only start to work in 2.4.0 (http://wowwiki.wikia.com/wiki/Patch_2.4.0):

> Weightstones and Sharpening Stones: The bonus damage from these temporary enchantments now works while shapeshifted.

**Comment by GuybrushGit** [comment]
>Don't get +damage either on a weapon... goes without saying I hope, but that won't do anything for you if you're not hitting with that weapon, even though it's not a true proc.
>http://wow.allakhazam.com/forum.html?forum=21&mid=113950310940335407

>Only thing that ever increased damage like weightstones and sharpening stones in animal form was that epic ring you get from Cenarion Circle.. I think they nerfed that though.. maybe still works.. don't remember. =)
>http://thedruidsgrove.org/archive/wow/t-12335.html

Only thing that might work for ferals is Might of Cenarius.
Multiple comments say it does:
https://web.archive.org/web/20070313193648/http://www.thottbot.com/i21189

Some say it does, some say it doesn't:
http://web.archive.org/web/20080603200517/http://wow.allakhazam.com:80/db/item.html?witem=21189

**Comment by Razorwynd** [comment]
Sad day for feral druids...

**Comment by haolerot** [comment]
Sorry, but both the +5dmg to 1h and +9dmg to 2h have always been bis for feral in retail Vanilla. This was changed in TBC.

The bug you were referencing that was fixed was the elemental damage on weapons being applied to feral forms.

**Comment by Fiskehatt** [comment]
Do you have any evidence to back that up? The sources referenced by me and GuybrushGit state otherwise. Moreover, here is a comment on the +9 weapon damage enchant posted around September 2006 (https://web.archive.org/web/20070515223857/http://www.thottbot.com:80/s20030):

> so is this helpful to feral druids playing as a cat, when their atk speed is set at 1 sec regardless of the weapons speed?

> Sadly not. This enchant has no effect on druids in feral forms. The damage range of your equipped weapon isn't carried over to your claws. 
>
> The good news is that the +15str enchant will benefit you, as will any other enchant that improves your stats.

**Comment by haolerot** [comment]
So let me get this straight, you're referencing comments of other random people as your solid proof. But my comment holds no weight? Will people look at my comment in 10 years as a source and say "See, +9 dmg worked in cat form."

**Comment by Fiskehatt** [comment]
Well, the common practice in proving blizzlike behaviour on this server seems to have been the referencing of old comments and videos from vanilla times: a lot of stuff on this server has been fixed solely based on them while ignoring comments from current players. However, I am not claiming that your comments hold no value, but compared to the number of comments from vanilla claiming that weapon damage enchants do not work, they do not seem very credible to me. Moreover, people now are likely to be more biased than back then (e.g. they cannot remember well the mechanics from over 10 years ago, have played on other private servers or later on retail wow, or just want to protect their classes or tactics). In 10 years, your comments will thus remain as credible as they are now.

Anyways, there is a chance that I might be wrong. Yet, I am not the one that you need to convince: the developers here decide whether to implement changes (blizzlike or not). My guess is that posting just your own opinion is not enough to convince them.

**Comment by haolerot** [comment]
So why are you posting this entire thing then, if you're so unsure of it being true. The weapon enchants 100% worked in vanilla. The Ring of Cenarius 100% worked in vanilla.

It's not a broken mechanic. Quit referencing the sole comments of a few selected idiots that probably didn't even play druids.

**Comment by Fiskehatt** [comment]
I did not say I was very unsure about it. I would rather say it seems more probable to me that they did not work in vanilla than did. That is enough for me to post this issue.

**Comment by haolerot** [comment]
Look at the comments that were linked regarding might of cenarius. People both stated that it did and did not work. That just shows the true idiocy of players at that time. All +weapon dmg enchants/buffs worked in feral forms.

**Comment by Razorwynd** [comment]
I am all for a more B-like experience... but this is something that is both: (1) unclear as to how it functioned a dozen years ago and (2) not particularly broken, problematic, or unfair (in fact it slightly enables an entirely different playstyle).  

Furthermore, many of the arguments as to why it should not function are absolute bullshit... Every time I see a stupid "druids don't use weapons they use their claws" comment, I reply with "Why does a bow with +1% hit make your melee attacks miss less often!  Why does a offhand dagger with 1% crit increase your crit chance with your bow!" 

This is the exact same behavior simply carried over to another character stat. If for no other reason than consistency, I think it should stay as it is ... From game design perspective, inconsistencies are far worse than non-intuitive behaviors.

**Comment by Fiskehatt** [comment]
> I am all for a more B-like experience... but this is something that is both: (1) unclear as to how it functioned a dozen years ago and (2) not particularly broken, problematic, or unfair (in fact it slightly enables an entirely different playstyle).

1. To me, the mechanic seems to be rather consistently described: all sources that I have found indicate that the weapon enchants did not work. There has only debate has been around Might of Cenarius. The bug has also been confirmed on [Kronos ](https://vanilla-twinhead.twinstar.cz/?issue=7647 )and old [Nostalrius](https://forum.nostalrius.org/viewtopic.php?f=41&t=36377&start=100#p287384) (where the users were even threatened with sanctions). Moreover, no one has yet to bring up any actual evidence that they worked. Is it not rather weird that no druids in vanilla noticed that they worked and reported it considering the large DPS boost?

2. I can agree that it is not currently very unfair or game-breaking. However, both insignificant and significant mechanics are constantly fixed here while seeking for a blizzlike experience. In addition, the current mechanic also causes another bug: the enchant damage is being applied for other classes even when they are disarmed or their weapon is broken, which does not really make sense.

> Furthermore, many of the arguments as to why it should not function are absolute bullshit... Every time I see a stupid "druids don't use weapons they use their claws" comment, I reply with "Why does a bow with +1% hit make your melee attacks miss less often! Why does a offhand dagger with 1% crit increase your crit chance with your bow!"
>
> This is the exact same behavior simply carried over to another character stat. If for no other reason than consistency, I think it should stay as it is ... From game design perspective, inconsistencies are far worse than non-intuitive behaviors.

No, there is a clear consistency: Might of Cenarius and for example Satyr's Bow have Equip-bonuses which grant you a bonus in general, whereas weapon damage enchants increase your weapon's damage, which has no impact on your DPS in feral forms or when disarmed/wearing a broken weapon.

Again, I encourage you to find evidence from vanilla sources to prove your arguments. Otherwise, this discussion leads nowhere.

**Comment by haolerot** [comment]
You find it weird that no druids talked about +9 dmg enchants? Find many threads about powershifting dps in vanilla? That's right, you wont. Just because there is a lack of people saying that it worked, doesn't mean it didn't exist. 

> Something I didn't know was that ferals get the benefits of Superior Impact (http://www.wowhead.com/?spell=20030) and the other "+damage to weapon" enchants. 
http://thedruidsgrove.org/archive/wow/t-12335.html

**Comment by Fiskehatt** [comment]
Powershifting was a known tactic in vanilla as well. For example, Tangedyn's DPS Cat fight simulator provided DPS calculations for both powershifting and non-powershifting:

http://i.pi.gy/rjpbm.png

Feels rather weird that he would have ever missed the weapon damage enchants. Moreover, you quote a source from 2008, not from vanilla times, which does not really prove anything as the patch notes indicated some changes in mechanics already in TBC. 

Also, lack of evidence does not naturally mean the mechanic did not exists, however, I have provided a wide range of evidence from vanilla stating the enchants did not work, whereas you have provided nothing useful to support your arguments.

**Comment by Razorwynd** [comment]
> Moreover, no one has yet to bring up any actual evidence that they worked. Is it not rather weird that no druids (not even the theorycrafters) in vanilla noticed that they worked and reported it considering the large DPS boost?
I agree it is weird, which probably indicates that it did not work. However, I can certainly remember druids testing this out. Nevertheless, I do not remember what the final answer was.

> I can agree that it is not currently very unfair or game-breaking. However, both insignificant and significant mechanics are constantly fixed here while seeking a blizzlike experience. In addition, the current mechanic also causes another bug: the enchant damage is being applied for other classes even when they are disarmed or their weapon is broken, which does not really make sense.
I know it is cliche, but B-like =/= B-exact. I think the current mechanics allow for a more immersive and enjoyable game play experience. Also, IIRC, recent changes to weapon skill, hit chance, and glancing blows, were not exactly B-exact, but rather B-inspired, and thus were introduced for other reasons.

> No, there is a clear consistency: Might of Cenarius and for example Satyr's Bow have Equip-bonuses which grant you a bonus in general, whereas weapon damage enchants increase your weapon's damage, which has no impact on your DPS in feral forms or when disarmed/wearing a broken weapon.

Fair point... and I presume the basic stats are presumed to be equip-bonuses as well?

**Comment by haolerot** [comment]
It 100% worked in retail vanilla. I had +5dmg on bestial fury. All sharpening stones worked in forms in vanilla. Only the adamantite crap that came out in BC didn't work at first, until later on in the xpac.

Please stop wasting the dev's time, by linking useless crap. Literally every single post you referenced, has both people saying something DOES work and then someone replying that it DOESN'T work.

**Comment by Fiskehatt** [comment]
>I know it is cliche, but B-like =/= B-exact. I think the current mechanics allow for a more immersive and enjoyable game play experience. Also, IIRC, recent changes to weapon skill, hit chance, and glancing blows, were not exactly B-exact, but rather B-inspired, and thus were introduced for other reasons.

Yeah, the mechanic makes ferals much more playable. However, many people have not opted for the enchant as using it was considered bug abuse on Nostalrius (and probably also here as Pottu originally banned it) and there was a risk of sanctions. This has basically divided druids into "exploiters" and non-exploiters ones, which is not really an optimal situation. Consequently, my original point in this creating this issue was just to bring it to the developer team's attention, allowing them to decide what to do about it. If they decide to do keep it as non-blizzlike, using the enchants would probably become acceptable. If they implement the changes, it again puts all druids on the same starting point. As the change is not live yet and not even on the dev server yet, we will have wait how they decide.

> Fair point... and I presume the basic stats are presumed to be equip-bonuses as well?

Basically yes, in cat form they affect your paws damage, which has nothing to do with your weapon damage and its damage enchants.

**Comment by haolerot** [comment]
I don't know why you're even bringing it up. The issue was already brought up on elysium github: the 3 bugged weapons, enchants, and stones. Dev's said they fixed the issue by fixing the bugged weapons. 

> This has basically divided druids into "exploiters" and non-exploiters ones, which is not really an optimal situation
please stfu with your drama. This is not happening anywhere. The same druids who raid as feral, still have +9dmg and +5dmg on their weapons.

> Basically yes, in cat form they affect your paws damage, which has nothing to do with your weapon damage and its damage enchants.

Both the Cenarion Ring and weapon enchants say "+4 Weapon Damage" and "+9 Weapon Damage." If the ring works, the enchants to weapon works as well. 

http://db.vanillagaming.org/?item=21189#wh-comments
> This is *the* best feral ring in the game. +4 weapon damage in cat form = +4 dps = +56 AP = ~23 strength with 5/5 HotW. By smcn on 10/23/2006 (Patch 1.12.2)

**Comment by Fiskehatt** [comment]
The ring is adds generic damage, not weapon damage, despite its tooltip. Enchants add weapon damage which has no effect in forms. Just look at GuybrushGit's comment:

>Might of Cenarius will continue working since its generic damage not weapon specific, despite what the tooltip says

**Comment by haolerot** [comment]
No, both the ring and weapon enchant add +Weapon Damage. It's not some random player's own interpretation of what the enchant does. They both specifically are worded the exact same way. +4 weapon damage from the ring adds 4 dmg to your autoattacks. +9/+5 from weapon enchants do the exact same thing. There's 0 confusion. 

Why would GuybrushGit's comment carry any weight? It states "+4 weapon damage." That sounds pretty fucking specific to weapon and not generic. You're implying that the tooltip is worded incorrectly? You'd think in the 10 years since the ring's implementation, blizzard would have fixed the tooltip on live servers. 

Please gtfo an

**Comment by llopinator** [comment]
Going to reiterate what haolerot said. Fiskehatt, you're dismissing the exact wording of a tooltip and interpreting it in an entirely different way. If there's anything that the devs should give credence to, it's the wording of a tooltip, not someone's unsupported insistence that "oh, no that tooltip that says +4 weapon damage - it's REALLY generic damage because trust me."

If Might of Cenarius' +4 weapon damage does indeed work for druids (and we know it does), then that's unquestionable precedent for +9 weapon damage enchants working as well. Weightstones? Perhaps not, since patch 2.4.0 explicitly makes weightstones work while shapeshifted, and it's unclear whether the weightstones (a) worked in Vanilla then didn't early TBC, or (b) never worked until 2.4.0. 

Might of Cenarius is precedent for +9 weapon damage enchant working for druids. This is a no brainer.

**Comment by Nostalian** [comment]
Going to reiterate what haolerot said. Fiskehatt, you're dismissing the exact wording of a tooltip and interpreting it in an entirely different way. If there's anything that the devs should give credence to, it's the wording of a tooltip, not someone's unsupported insistence that "oh, no that tooltip that says +4 weapon damage - it's REALLY generic damage because I say so."

If Might of Cenarius' +4 weapon damage does indeed work for druids (and we know it does), then that's unquestionable precedent for +9 weapon damage enchants working as well. Weightstones? Perhaps not, since patch 2.4.0 explicitly makes weightstones work while shapeshifted, and it's unclear whether the weightstones (a) worked in Vanilla then didn't early TBC, or (b) never worked until 2.4.0.

Might of Cenarius is precedent for +9 weapon damage enchant working for druids. This is a no brainer.

**Comment by Nostalian** [comment]
absent having the actual vanilla code - not LH core code - there's no evidence that the +4 weapon damage from might of cenarius was meant to be applied to some passive weapon damage value. one could just as easily interpret +4 weapon damage as increasing the damage of whatever weapon the druid currently has equipped, a case in which clearly the druid's 'claws' wouldn't qualify as a weapon because they are not equipped by the druid per se(edited)
the burden of proof is and should be on the nay-sayers to convincingly show that might of cenarius functioned uniquely in this  way during vanilla. until that happens, and if we buy that might of cenarius worked for druids in vanilla (which most seem to do), it's sufficient precedent for +9 wpn dmg working as well

**Comment by ZakPreston** [comment]
Is there any single proof that +X Weapon damage enchants didn't work in Vanilla for druids (except a couple of comments from wowhead)? This whole fuss with "fixing" weapon damage for cats and bears looks like a voluntaristic attempt to fix something for the sake of fixing. 
But the real problem is that by fixing the questionable game mechanics you are literally making one of the worst specs even less playable one. I'm all for blizzlike changes, but only 100% confirmed ones, sorry.

**Comment by supasteed** [comment]
@Nostalian theres also proof on getbuffeddata that might of cenarius does work for druids which is weapon damage

Its on the blazefury medallion page

**Comment by supasteed** [comment]
http://wowdata.buffed.de/item/Ruestung/Verschiedenes/Medaillon-des-lodernden-Zorns-17111

This page in n comments indicates might of cenarius works for druids

**Comment by AliUDmov** [comment]
Stop the witch hunt and confess to the truth. There is not a single source 2004-2006 that +weapon damage enchants worked in Vanilla, as well as Weightstones. There are dozens of sources that state it didn't work.

You are just pissed that you lose some dps. Just get the AQ40 mace with 280 Attack Power and ignore the whole weapon damage abuse.
btw... you can get banned when you are reported repeatedly for abusing the enchant and Weightstones. So don't act like you are honest feral druids.

**Comment by haolerot** [comment]
There's numerous _comments_ that state that band of cenarius doesn't work, yet even MORE stating it does in fact work.
These are the same idiots saying that +weapon dmg enchants/stones to weapons don't work, so you're taking that as proof?

If the +weapon dmg ring works the +weapon dmg enchant works as well. They're identical buffs.

**Comment by AliUDmov** [comment]
> If the +weapon dmg ring works the +weapon dmg enchant works as well. They're identical buffs.

Sorry but this is not correct.

Might of Cenarius applies the SPELL_AURA_MOD_DAMAGE_DONE modifier. You find it here: http://db.vanillagaming.org/?spell=25901
Weightstones and weapon damage enchants do not apply a SPELL_AURA_MOD_DAMAGE_DONE modifier.

Compare it to weapon effects (chance on hit) and trinket effects (chance on hit). They should only work for trinket effects, but not for weapon effects. That's how it worked back in the days. 

And here is another source for that: http://thedruidsgrove.org/archive/wow/t-11401.html

**Comment by haolerot** [comment]
You’re an idiot. Did you even read the entire post you linked? The amount of uncertainty in that thread is overwhelming. 

It’s not a chance on hit. It’s adding weapon damage to your physical attacks, it’s not hard to grasp. People have stated that the elemental sharpening stones don’t work, yet someone in the thread you linked state that they do. 

“The highest level stone that increased the chance to crit worked for druids.
Well technically its not a weapon proc, more like "enchanting" your weapon with +agi or +2% crit.” 

Unless you can come up with solid, definitive proof that the stones/enchants didn’t work, other than a bunch of uncertain plebs on a BC forum, then no changes should be made on how these enchants/stones currently work.

**Comment by AliUDmov** [comment]
You are the idiot lol

the ring worked
elemental sharpening stones worked
stat enchants worked (15 agi etc)

on hit effects did not work
+damage weightstones did not work
+damage enchants did not work

there are no sources that speak against this...
the linked forum ist vanilla, too.

**Comment by haolerot** [comment]
Please sit down. The post you linked is from 12/21/06. 

Patch 2.0 was 12/5/06

**Comment by AliUDmov** [comment]
there are other posts that are a lot older and suggest the same

**Comment by AliUDmov** [comment]
Fiskehatt did a great job with finding evidence here: https://github.com/LightsHope/server/pull/371

There is no evidence that the bugged state on LH is blizzlike. You failed to deliver it. The bug will be fixed due to facts. 
Insulting people as idiots will not change anything for you.

**Comment by haolerot** [comment]
Also Fiskehatt, please don’t use Tangedyn as a point of reference. He didn’t even powershift. I do more damage than him, being bwl geared,  compared to him being naxx geared.
